### PR TITLE
Truncate large numbers in beatmap stats

### LINKED ISSUE #1879: Avoid beatmap data overflowing off the page
https://osu.ppy.sh/beatmapsets/701416#mania/1484517
screen shot 2017-12-03 at 5 09 14 pm

### PR DESCRIPTION
**Pull request by TPGPL**
Resolves #1879

---

### CODE REVIEW

File: resources/assets/coffee/osu_common.coffee [hunk 1]
```diff
@@ -214,7 +214,7 @@
 
 
   formatNumberSuffixed: (number, precision, options = {}) ->
-    suffixes = ['', 'k', 'm', 'b', 't']
+    suffixes = ['', 'k', 'm', 'b', 't', 'qa', 'qi', 'sx', 'sp', 'oc', 'no']
```
Comment (peppy) [inline resources/assets/coffee/osu_common.coffee]: what the fuck

Comment (bdach) [reply resources/assets/coffee/osu_common.coffee]: I think these new ones are supposed to stand for `quadrillion`, `quintillion`, `sextillion`, ..., `nonillion`, but yeah, these don't seem to be standardised in any way, so probably shouldn't be used. (Also are numbers that large even showing up anywhere?)

Comment (TPGPL) [reply resources/assets/coffee/osu_common.coffee]: the issue requires at least quintillion in the suffixer, added more as some beatmaps might have even bigger numbers, can delete some if thats a problem

Comment (cl8n) [reply resources/assets/coffee/osu_common.coffee]: maybe add an option to default to infinity symbol if it's beyond trillions

Comment (peppy) [reply resources/assets/coffee/osu_common.coffee]: I like the infinity idea, if that's feasible (would need to hide the number itself too).

Comment (TPGPL) [reply resources/assets/coffee/osu_common.coffee]: i could go with the infinity char if its not considered as a workaround, the original BPM will still be shown in a tooltip

**Pull request merged**